### PR TITLE
feat: Docs - test types updated

### PR DESCRIPTION
**Pull request by tkonieczny**

### CODE REVIEW

**Review by vsukhin — CHANGES_REQUESTED** [review]
remerge main branch

File: docs/docs/test-types/container-executor.md [hunk 1]
```diff
@@ -1,9 +1,92 @@
...
+![Container executor settings - Command and Arguments](../img/dashboard-executor-settings-command-arguments.png)
+
+Container image tab allow you to change the container image which is used, or set `Secret ref name` if the image is in the private registry.
+Additionally, the Definition tab allows you to check the Executor CRD:
+
+![Container executor settings - Definition](../img/dashboard-executor-settings-definition.png)
+
+Now, the container executor is created, and configured - it can be used for running tests.
+
+## Testkube CLI
```
Comment (aabedraba) [inline docs/docs/test-types/container-executor.md:34]:
```suggestion
### Testkube CLI
```

File: docs/docs/test-types/container-executor.md [hunk 2]
```diff
@@ -1,9 +1,92 @@
...
+
+
+- `--name` is mandatory - that's custom executor's name.
+- `--executor-type` need to be set to `container`
+- `--image` is the executor image (for example, `curlimages/curl:7.85.0`)
+- `--types` that's the name you will later use in your Tests as the test type (for example, `executor-curl/test`)
+- `--command` (for example, `curl`)
+
+Which result in:
+```
```
Comment (aabedraba) [inline docs/docs/test-types/container-executor.md:83]:
```suggestion
```sh
```

File: docs/docs/test-types/container-executor.md [hunk 3]
```diff
@@ -1,9 +1,92 @@
...
+
+![Container executor settings - Definition](../img/dashboard-executor-settings-definition.png)
+
+Now, the container executor is created, and configured - it can be used for running tests.
+
+## Testkube CLI
+Custom Container Executor can also be created with Testkube CLI using `testkube create executor` command:
+
+Let's check available configuration options first:
+```
```
Comment (aabedraba) [inline docs/docs/test-types/container-executor.md:38]:
```suggestion
```sh
```

**Review by aabedraba — APPROVED** [review]

**Review by vsukhin — APPROVED** [review]

**Pull request merged**